### DISCUSSION TITLE
Why is deadcode not deleted?

### DISCUSSION THREAD

**Discussion by JoshInnis** [heterodb/pg-strom#693] [General]
I am browsing the codebase right now, and wondering why code gets moved to a seperate directory rather than removed outright? Is this to preserve history in a way that is easy to find the old files?

**Comment by kaigai** [comment]
Because I'm not 100% certain whether we want to reuse / reference the previous code & logic in the future.
From practical standpoints, you can ignore the contents of deadcode directory.
